.github/workflows/e2e.yaml

```diff
@@ -16,13 +16,19 @@ jobs:
       with:
         go-version: '~1.17'
     - name: Install kind
+      env:
+        KIND_VERSION: v0.11.1
+        KIND_OS: kind-linux-amd64
       run: |
-        curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
         chmod +x kind
         sudo mv kind /bin/
     - name: Create kind cluster
       run: |
+        kind version
         kind create cluster
         kind export kubeconfig
+    - name: Install Ginkgo
+      run: go install github.com/onsi/ginkgo/ginkgo@v1.16.4
```

Comment:
It looks like this got added to the tools.go, so this should be necessary anymore?

Reply:
Just tested this now and it this line appears necessary to install the Ginkgo CLI used in `make test-e2e`.

Reply:
I think that's because we can't use the path@version syntax in tools.go like we can with go get/go install, so we need to remove any direct versioning through tools.go, and instead handle any version requirements through the root go.mod.